GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -289,6 +289,7 @@ private ContextMenuSelectionInfo GetSelectionInfo()
             bool isAnyWorkTree = DiffFiles.SelectedItems.Any(item => item.Staged == StagedStatus.WorkTree);
             bool isAnySubmodule = DiffFiles.SelectedItems.Any(item => item.IsSubmodule);
             bool allFilesExist = DiffFiles.SelectedItems.All(item => File.Exists(_fullPathResolver.Resolve(item.Name)));
+            bool allFilesOrDirectoryExist = allFilesExist || DiffFiles.SelectedItems.All(item => FileOrUntrackedDirExists(item));
```

Comment:
These two states should be obtained by one merged function.

BTW: How to create an "untracked dir"?

Reply:
See screenshot, just copy a submodule

Reply:
Do you want to use a for loop to get the values then?

Reply:
> Do you want to use a for loop to get the values then?

Yes, I think this is better than resolving the path and check `File.Exists` twice.